src/repl.ts

```diff
@@ -30,7 +30,7 @@ export interface ReplService {
    * Bind this REPL to a ts-node compiler service.  A compiler service must be bound before `eval`-ing code or starting the REPL
    */
   setService(service: Service): void;
-  evalCode(code: string): void;
+  evalCode(code: string): { awaitPromise: boolean; result: any };
```

Comment:
Is there a way to avoid breaking changes to our public API surface, perhaps by adding a new method?

Reply:
`evalCode` could have an optional second argument, and based on that it returns an string or an object (the default being the string to avoid being a breaking change), else another method would need to be used